examples/tutorials/basics/polygons.py

```diff
@@ -0,0 +1,106 @@
+"""
+Plotting polygons
+=================
+
+Plotting polygons is handled by the :meth:`pygmt.Figure.plot` method.
+
+This tutorial focuses on input data given via lists or NumPy arrays. For
```

Comment:
> via lists or NumPy arrays

I feel we should just use NumPy arrays in this example, and mention that any array-like objects are accepted.

Reply:
OK with using both times NumPy arrays. Code and documentation are updated in commit https://github.com/GenericMappingTools/pygmt/pull/3593/commits/bd71d3e835ed6a9d731e1c748261f94186bee952.